e2etests/e2e_test.go

```diff
@@ -209,6 +209,7 @@ func run(t *testing.T, tc testCase) {
 		renderOpts := &d2svg.RenderOpts{
 			Pad:     0,
 			ThemeID: tc.themeID,
+			// SetDimensions: true,
```

Comment:
why?

Reply:
I was checking the actual dimensions vs measured dimensions, removed this commented out line.